extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeStreamExtractorDASHText.java

```diff
@@ -0,0 +1,60 @@
+package org.schabi.newpipe.extractor.services.youtube;
```

Comment:
You shouldn't have pulled my changes, you should have re based them, because now my commits are part of this branch :/, see [here](https://github.com/TeamNewPipe/NewPipeExtractor/pull/100/commits).

Reply:
Thats a lower issue though, the rest looks fine :)

Reply:
@theScrabi yeah I noticed.
Is there a chance to fix this now without breaking it?
Would be great if you could help me with this :)

Reply:
You could try to remove that commit by doing an interactive rebase, but thats pain.

Reply:
I could remove the changes from this commit by hand, can't I?

Reply:
or make a soft reset before your commit and the stash onto that?